src/peripheral/scb.rs

```diff
@@ -300,7 +301,7 @@ impl VectActive {
             12 => VectActive::Exception(Exception::DebugMonitor),
             14 => VectActive::Exception(Exception::PendSV),
             15 => VectActive::Exception(Exception::SysTick),
-            irqn if irqn >= 16 => VectActive::Interrupt { irqn },
+            irqn if irqn >= 16 && irqn < 512 => VectActive::Interrupt { irqn: irqn - 16 },
```

Comment:
the `- 16` was added to be consistent with how `vect_active` does it

Reply:
Is that a breaking change?

Reply:
Yes, but I think the rest of this PR is too